frontend/packages/ceph-storage-plugin/integration-tests/views/add-capacity.view.ts

```diff
@@ -0,0 +1,11 @@
+import { $ } from 'protractor';
+
+export const ocsOp = $('a[data-test-operator-row="Openshift Container Storage Operator"]');
+export const storageClusterView = $(
+  'a[href="/k8s/ns/openshift-storage/operators.coreos.com~v1alpha1~ClusterServiceVersion/ocs-operator.v0.0.2/ocs.openshift.io~v1~StorageCluster"]',
+);
```

Comment:
This is going to break. Use a different approach. Look at my PR.

Reply:
can you point the change here?

Reply:
v.0.0.2 is gonna change. previously it was 0.0.01. Navigate through the UI.

Reply:
@bipuladh Even you have hardcoded the child to 10 which again can be broken.

Reply:
https://github.com/openshift/console/blob/master/frontend/packages/ceph-storage-plugin/integration-tests/views/installFlow.view.ts#L13 - is this you referring to?

Reply:
Lesser chances than a change in version number. Unless they introduce new pieces to operator. But I guess we can let this one be.

Reply:
Atleast use CONST instead of these hardcoded values. Makes it easier to change later if required.